### PR TITLE
Feature/blog List

### LINKED ISSUE #19: TODO: Blog detail

### PR DESCRIPTION
**Pull request by buitannguyen2612**
closes #19

### CODE REVIEW

File: src/components/CardBlog/CardBlog.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        return () => {
+            searching = false
+        }
+    }, [valueSearch])
+
+    // xử lý các giá trị scroll
+    const handleScroll = () => {
+        positions.scroll = window.pageYOffset
+        positions.sectionHeight = getPosition.current.clientHeight
+        positions.topContent = getPosition.current.offsetTop + 272
```
Comment (phunlh2001) [inline src/components/CardBlog/CardBlog.jsx]: What is the number `272`? You should create a variables for this number easy-to-read.

File: src/routes/index.js [hunk 2]
```diff
@@ -17,6 +18,7 @@ const publicRoutes = [
     { path: '/detail', page: DetailPage, layout: MainLayout },
     { path: '/about-us', page: 'about us here', layout: MainLayout },
     { path: '/predict', page: DetectionPage, layout: MainLayout }, // cai nay de test, sau nay chuyen ve privateRouter
+    { path: '/bloglist', page: BlogListPage, layout: MainLayout }, // cai nay de test, sau nay chuyen ve privateRouter
```
Comment (phunlh2001) [inline src/routes/index.js:24]: we have a route `/blog` before. You can use it instead of this one.

**Review by phunlh2001 — CHANGES_REQUESTED** [review]
Please fix these. 
Besides, I see a lot of warning about your states are never used, format the import never used with `Alt + Shift + O` and deleting every states you never used it, please.

File: src/routes/index.js (hunk 2 above)
Comment (buitannguyen2612) [reply src/routes/index.js:24]: anything wrong right? the blog path is belong to blogPage, so its still good if we have 2 component in 1 path

**Pull request merged**